modules/nextMillenniumBidAdapter.js

```diff
@@ -97,7 +119,9 @@ export const spec = {
           contentType: 'application/json',
           withCredentials: true
         },
-        bidId: bid.bidId
+
+        bidId,
+        auctionId,
```

Comment:
The `postBody` only supports the `banner` mediaType and not `video`.  Video params must be read from `AdUnit.mediaTypes.video` when constructing a request for video.

Reply:
@lksharma Yes. Empty size is expected case in our backend